### PR TITLE
removes user card on delete without refreshing / redirecting

### PR DESCRIPTION
**Pull request by JacksonBates**
This should reactify the delete button.

@jenovs - could you review when you get time?

### CODE REVIEW

File: src/Campers.js [hunk 1]
```diff
@@ -8,11 +9,28 @@ class Campers extends Component {
...
+                   techSetup: Array.isArray(setup) ? setup.join(', ') : setup,
+                   id: this.props.camper_id,
+                   display: true };
+  
+    this.handleSubmit = this.handleSubmit.bind(this);
   }
   
+  handleSubmit(e) {
+    e.preventDefault();
+    const url = 'https://enigmatic-dawn-95873.herokuapp.com/api/v1/posts/' + this.props.camper._id;
```
Comment (JacksonBates) [inline src/Campers.js:21]: This is using the new delete route - we can probably get rid of the redundant ones once this merges

Comment (jenovs) [reply src/Campers.js:21]: As you merged my mock server code, the server URL here needs to be replaced with server variable. 
And yes the old routes can be removed or at least commented out.

Comment (JacksonBates) [reply src/Campers.js:21]: Is the server variable in the App.js? We can pass it as props, right? (Still figuring out React...)

(I'll come back to this later - gotta take the kid to a Playgroup Christmas thing :) )

Comment (jenovs) [reply src/Campers.js:21]: Look at the merged code in Campers.js - import server from config at the top and replace `http...` with `${server}` in URL.

Comment (JacksonBates) [reply src/Campers.js:21]: Oh, I see. That's why there's a merge conflict. Easy enough to fix :)

**Pull request merged**